src/views/clusteroverview/OverviewTab/vm-statuses-card/VMStatusItem.tsx

```diff
@@ -28,7 +34,14 @@ const VMStatusItem: React.FC<VMStatusItemProps> = ({ count, namespace, status })
         <div className="vm-statuses-card__status-item--count">
           <span className="vm-statuses-card__status-item--icon">{Icon && <Icon />}</span>
           <span className="vm-statuses-card__status-item--value">
-            <Link to={path}>{count.toString()}</Link>
+            <Link
+              onClick={() => {
+                onFilterChange();
+              }}
```

Comment:
```suggestion
              onClick={onFilterChange && () => {
                onFilterChange();
              }}
```

`onFilterChange` is optional. Does it crash when the user clicks on the link when no method is defined?